<?xml version="1.0"?>
<launch>
	<arg name="port" default="/dev/ttyUSB1" />
	<arg name="frame_id" default="laser" />

	<node pkg="robofriend" type="RobobrainNode.py" name="robofriend_robobrain" output="screen">
	</node>

  	<node pkg="hls_lfcd_lds_driver" type="hlds_laser_publisher" name="hlds_laser_publisher" output="screen">
		<param name="port" value="$(arg port)"/>
		<param name="frame_id" value="$(arg frame_id)"/>
	</node>

	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find hls_lfcd_lds_driver)/rviz/hlds_laser.rviz" />

	<!-- to be implemented -->
	<node pkg="robofriend" type="GameCommunicatorNode.py" name="robofriend_game_communicator" output="screen">
	</node>

	<node pkg="robofriend" type="RFIDNode.py" name="robofriend_rfid_handler" output="screen">
	</node>

	<node pkg="robofriend" type="PCBSensorData.py" name="robofriend_pcb_sensor_data" output="screen">
	</node>

	<node pkg="robofriend" type="TeensyNode.py" name="robofriend_teensy_communicator" output="screen">
	</node>
</launch>
